.github/scripts/is-published.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# for the name and version specified in package.json
+pkg=$(node -e "console.log(require('./package.json').name)")
+version=$(node -e "console.log(require('./package.json').version)")
```

Comment:
Minor, might be useful to see the package version echoed out here to quickly identify which binary was unpublished.

Reply:
Done.